actioncable/lib/action_cable.rb

```diff
@@ -35,7 +37,15 @@ module ActionCable
       confirmation: 'confirm_subscription'.freeze,
       rejection: 'reject_subscription'.freeze
     },
-    default_mount_path: '/cable'.freeze
+    default_mount_path: '/cable'.freeze,
+    # It is important that the unsupported protocol be last in the list,
+    # as the Websocket Driver always chooses the first protocol that matches,
+    # with the intended result that if the client sends the unsupported
+    # protcol as an option, it will be the last chosen.
```

Comment:
Ok I did that. Take a look. I think that looks much cleaner and makes it so nobody really has to think about the ordering.